README.md

```diff
@@ -2,9 +2,9 @@
 
 Quickly view GitHub issues for the currently opened project.
 
-![](https://r24y.github.io/quick-issues/docs/issues-list.png)
+![](./docs/issues-list.png)
```

Comment:
Unfortunately, these need to remain absolute URLs pointing at GitHub Pages; AFAIK, otherwise the images won't show up on the atom.io package listing, or in Atom's **Settings** view.